### PR TITLE
Job mode triggering

### PR DESCRIPTION
**Pull request by kkuehlz**
Fixes: #3043

### CODE REVIEW

File: man/systemctl.xml [hunk 1]
```diff
@@ -1647,6 +1648,13 @@ Jan 12 10:46:45 example.com bluetoothd[8900]: gatt-time-server: Input/output err
         dependencies will still be honored.</para>
         </listitem>
 
+        <para>If <literal>triggering</literal> is specified with
+        <command>systemctl stop</command>, the target unit is stopped
+        and any active units that trigger the target units are stopped.
+        This mode only works with <command>systemctl stop</command>. For
+        Triggers=, see
+        <citerefentry><refentrytitle>systemd.unit</refentrytitle><manvolnum>5</manvolnum></citerefentry>.</para>
```
Comment (keszybz) [inline man/systemctl.xml]: The first two sentences repeat the same thing. Also Triggers= should use `<varname>`.
And "target" has a special meaning in the context of units, so it should not be used here.
Maybe
```xml
<literal>triggering</literal> may only be used with <command>systemctl stop</command>.
In this mode, the specified unit and any active units that trigger it are stopped.
See the discussion of <varname>Triggers=</varname> in
<citerefentry><refentrytitle>systemd.unit</refentrytitle><manvolnum>5</manvolnum></citerefentry>
for more information about triggering units.
```

File: man/systemctl.xml [hunk 2]
```diff
@@ -1601,8 +1601,9 @@ Jan 12 10:46:45 example.com bluetoothd[8900]: gatt-time-server: Input/output err
         <literal>replace-irreversibly</literal>,
         <literal>isolate</literal>,
         <literal>ignore-dependencies</literal>,
-        <literal>ignore-requirements</literal> or
-        <literal>flush</literal>. Defaults to
+        <literal>ignore-requirements</literal>,
+        <literal>flush</literal> or
```
Comment (keszybz) [inline man/systemctl.xml]: I'd prefer the oxford comma here: "flush, or triggering".

File: src/core/manager.c [hunk 3]
```diff
@@ -1740,6 +1740,9 @@ int manager_add_job(
         if (mode == JOB_ISOLATE && !unit->allow_isolate)
                 return sd_bus_error_setf(error, BUS_ERROR_NO_ISOLATION, "Operation refused, unit may not be isolated.");
 
+        if (mode == JOB_TRIGGERING && type != JOB_STOP)
+                return sd_bus_error_setf(error, SD_BUS_ERROR_INVALID_ARGS, "Triggering is only valid for stop.");
```
Comment (keszybz) [inline src/core/manager.c]: Maybe `--mode=triggering is only ...`. This error message might be unclear without context.

**Comment by dtardon** [comment]
@keur Could you add "Fixes: #3043" to commit message?

**Comment by keur** [comment]
Changes made @keszybz

**Pull request merged**